yarn-project/end-to-end/src/e2e_2_rpc_servers.test.ts

```diff
@@ -191,5 +204,43 @@ describe('e2e_2_rpc_servers', () => {
 
     const storedValue = await getChildStoredValue(childCompleteAddress, aztecRpcServerB);
     expect(storedValue).toBe(newValueToSet);
-  }, 60_000);
+  });
+
+  it('private state is "zero" when Aztec RPC Server does not have the account private key', async () => {
+    const userABalance = 100n;
+    const userBBalance = 150n;
+
+    const completeTokenAddress = await deployTokenContract(userABalance, userA.address);
+    const contractWithWalletA = await TokenContract.at(completeTokenAddress.address, walletA);
+
+    // Add account B to wallet A
+    await aztecRpcServerA.registerRecipient(userB);
+    // Add account A to wallet B
+    await aztecRpcServerB.registerRecipient(userA);
+
+    // Add token to RPC server B
```

Comment:
nit: add a note that token is already added to RPC server A

Reply:
Addressed in [9ffb3a5](https://github.com/AztecProtocol/aztec-packages/pull/2491/commits/9ffb3a5c08ed15391d5f941f7b8a03044b95a5b0)